v03_pipeline/bin/pipeline_worker.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import datetime
+import os
+import time
+
+import luigi
+
+from v03_pipeline.api.model import LoadingPipelineRequest
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.paths import (
+    loading_pipeline_queue_path,
+    project_pedigree_path,
+    project_remap_path,
+)
+from v03_pipeline.lib.tasks import (
+    UpdateCachedReferenceDatasetQueries,
+    UpdateVariantAnnotationsTableWithNewSamplesTask,
+    WriteProjectFamilyTablesTask,
+)
+
+logger = get_logger(__name__)
+
+
+def main():
```

Comment:
The idea is for this process to run as a sidecar in the same pod, basically looping indefinitely for a request to show up, then blocking on doing the work, then releasing the request once it succeeds or finishes.